app/screens/postings/screen.js

```diff
@@ -15,6 +15,10 @@ class PostingListScreen extends Component {
     render() {
         return <PostingList {...this.props}/>
     }
+
+    componentDidMount() {
+        this.props.fetchNewPostings();
```

Comment:
soooo we weren't loading them before? or were we loading them in another component?

Reply:
will have a look into this, just noticed the blank screen after downloading the app